app/controllers/staff/organisations_controller.rb

```diff
@@ -4,7 +4,7 @@ class Staff::OrganisationsController < Staff::BaseController
   include Secured
 
   def index
-    @organisations = Organisation.all
+    @organisations = current_user.organisations
```

Comment:
If we make this change every user can only see those that they themselves are associated with. I think from dicussions so far that `administrators` and `fund managers` will be able to manage all users and organisations.

Can we do this in such a way that the future administrator role can see all organisations?

Reply:
I have take nit out for now but left the tests in (skipped). I think this should be handled with roles?

Reply:
Yeah, this can be done with a Pundit scope.